Assets/BossRoom/Scripts/DebugCheats/DebugCheatsManager.cs

```diff
@@ -121,6 +121,26 @@ void ToggleGodModeServerRpc(ServerRpcParams serverRpcParams = default)
             }
         }
 
+        [ServerRpc(RequireOwnership = false)]
+        void HealPlayerServerRpc(ServerRpcParams serverRpcParams = default)
+        {
+            var clientId = serverRpcParams.Receive.SenderClientId;
+            var playerServerCharacter = PlayerServerCharacter.GetPlayerServerCharacter(clientId);
+            if (playerServerCharacter != null)
+            {
+                var baseHp = playerServerCharacter.NetState.CharacterClass.BaseHP.Value;
+                if (playerServerCharacter.NetState.LifeState == LifeState.Fainted)
+                {
+                    playerServerCharacter.Revive(null, baseHp);
+                }
+                else
+                {
+                    playerServerCharacter.ReceiveHP(null, baseHp);
+                }
```

Comment:
Shouldn't this be removed now since the targeted kills are done on another cheat?

Also, modify PR name, description, etc to reflect it being for heals.

Reply:
This isn't the kill player cheat, with a positive value, ReveiveHp heals. This is to bring players back to full health if they haven't fainted. But yeah I'll update the PR

Reply:
Ah of course. So revive if dead and completely heal otherwise. Makes sense.